src/main/java/com/kodilla/ecommercee/controller/product/ProductController.java

```diff
@@ -0,0 +1,42 @@
+package com.kodilla.ecommercee.controller.product;
+
+import com.kodilla.ecommercee.domain.Product.ProductDto;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/product")
+public class ProductController {
+
+    @RequestMapping(method = RequestMethod.GET, value = "getProducts")
+    public List<ProductDto> getProducs(){
+        List<ProductDto> products = new ArrayList<>();
+        products.add(new ProductDto(1L, "testDescription", new BigDecimal(10)));
+        products.add(new ProductDto(2L, "testDescription2", new BigDecimal(10)));
+        return products;
+    }
+
+    @RequestMapping(method = RequestMethod.GET, value = "getProduct")
+    public ProductDto getProduct(@RequestParam Long productId) throws ProductNotFoundException{
```

Comment:
rozumiem, że @requestbody to tak na przyszłość? bo na razie mieliśmy zrobić kontrolera zwracającego sztuczne dane więc to nie było konieczne.

Reply:
tak to propozycja juz pracuję nad rzeczywistą implementacją tego wiec zobaczymy jak wyjdzie